docs/contributing.md

```diff
@@ -0,0 +1,17 @@
+First of all, welcome! Thanks for even reading this page. If you're here, you're probably wondering what you can do to help make the libsass project even more awesome. And, even having that feeling means you are awesome!
+
+## I'm a programmer
+
+Awesome! We need your help. The best thing to do is go find issues that are tagged with both "bug" and "test written". We do spec driven development here and these issues have a test that's written already in the sass-spec project. Go find the test by going to sass-spec/spec/libsass-todo-issues/issue_XXX/ where XXX is the issue number. Write the code, and compile, and then issue a pull request referencing the issue. We'll quickly verify it and get it merged in!
+
+To get your dev environment setup, checkout our article on [Setup-Dev-Environment](setup-environment.md).
+
+## I'm not a backend programmer
+
+COOL! We also need your help. Doing [Issue-Triage](triage.md) is a big deal and something we need constant help with. That means helping to verify issues, write tests for them, and make sure they are getting fixed. It's being part of the smiling face of the project.
+
+Also, we need help with the Sass-Spec project itself. Just people to organize, refactor, and understand the tests in there.
+
+## I don't know what a computer is?
+
+Hmm.... well, it's the thing you are looking at right now. Ummm... checkout training courses! Then, come back and join us!
```

Comment:
I'm not sure the joke's strong enough to stand on its own, but if you keep this section, "check out" would be two words here as well.